importer.js

```diff
@@ -20,18 +21,22 @@ fs.readFile('./users.csv', 'utf-8').then(async data => {
     await page.type(process.env.pw);
     await page.click('button[type="submit"]');
 
-    setTimeout(async cans => {
-      data.split(/\r|\n/gi).forEach(async studentID => {
-        await page.goto('https://google.com', {waitUntil: 'networkidle'});
-        await page.goto('https://websvc.dcu.ie/clubs/socs/register', {waitUntil: 'networkidle'});
-        await page.waitForSelector('#form_id');
-        await page.click('#form_id');
-        await page.type(studentID);
-        await page.click('button[type="submit"]');
-        await page.waitForSelector('input[type="checkbox"]');
-        await page.click('input[type="checkbox"]');
-        await page.click('button[type="submit"]');
+    setTimeout(() => {
```

Comment:
why do you use timeout?

Reply:
To make sure the auth chain finishes

Reply:
To make sure the auth chain finishes

Reply:
why not use the wait [function](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagewaitforselectorselector-options)